model/strand.rb

```diff
@@ -36,9 +36,10 @@ def self.prog_verify(prog)
   end
 
   def self.lease(id)
+    schedule = Config.development? ? "(now() + least(5, try) * '1 second'::interval)" : "(now() + least(2 ^ least(try, 20), 600) * random() * '1 second'::interval)"
```

Comment:
How about putting this into a const, so it's computed exactly once at start-up?

Reply:
Makes sense. I did that. Funnily, it made testing significantly difficult as that line would be executed at load time. So I added :nocov:. Any suggestions to test it?

Reply:
We have encountered the same issue in several places. The only solution I found for reloading constant values was by using ﻿`load "FILEPATH"`. Consequently, we decided to add ﻿`:nocov:` to them.

Reply:
I think nocov is fine for this case. Automation of coverage inspection is more important than the branch coverage per se.